File: Programming-languages-and-laboratory/Java/Exercises/29_compito_java_2019_04_11b/Sommelier.java
public class Sommelier extends Impiegato {

	private int ore;
	private String spec;

	public Sommelier(int codice, String nome, int ore, String spec, double costo) {
		super(codice, nome, "sommelier", costo);
		this.ore = ore;
		this.spec = spec;
	}

	public String toString() {
		return super.toString() + "\t" + ore + "\t" + spec + "\t-\t-\t-\t" + getCosto();
	}

}

File: II - Linguaggi di programmazione e laboratorio/Python/Exercises/29b_compito_java_20190411/Sommelier.py
from Impiegato import Impiegato


class Sommelier(Impiegato):
    def __init__(self, codice, tipo, nome, ore, costo, specialita):
        super().__init__(codice, tipo, nome, costo)
        self._ore = ore
        self._specialita = specialita

    def __str__(self):
        return super().__str__() + "\tsommelier\t" + str(self._ore) + "\t" + self._specialita + "\t-\t-\t-\t" + \
            str(self._costo)
